src/client/components/app/AmountInput/index.tsx

```diff
@@ -32,17 +32,22 @@ const StyledInput = styled.input<{ readOnly?: boolean; error?: boolean }>`
   ${({ readOnly, theme }) =>
     readOnly &&
     `
-    border: 1px solid ${theme.colors.textsVariant};
-    color: ${theme.colors.textsVariant};
+    border: 1px solid ${theme.colors.txModalColors.onBackgroundVariantColor};
+    color: ${theme.colors.txModalColors.onBackgroundVariantColor};
     cursor: default;
     background: transparent;
 
     &::placeholder {
-      color: ${theme.colors.textsVariant};
+      color: ${theme.colors.txModalColors.onBackgroundVariantColor};
```

Comment:
looking at the light color theme, seem they were using the correct color... I suppose you were testing with dark theme? Ideally both should use the same property, but if they are not consistent, we probably need to add a new one so that they match...

Reply:
Good catch, created an input placeholder prop in the theme and used it there

Reply:
https://github.com/yearn/yearn-finance-v3/pull/730/commits/892c876e61e2116537469a164a77223d34a854c1 & https://github.com/yearn/yearn-finance-v3/pull/730/commits/80bf8a01fc985a6457fb0036d84f3a6f1638b22f